src/job_description_feature_extraction.py

```diff
@@ -56,6 +60,58 @@ def get_feature_word_count(column_to_count):
     return count_dict
 
 
+def get_tfidf_similarity(corpus, queries):
+    """
+    calculate cosine similarity using tf.if between corpus an queries
+
+    :param corpus: list of text
+    :param queries: list of text
+    :return: matrix of ONLY corpus x query similarities
+    """
+
+    # # easier to test with smaller data set
+    # # use this to overwrite the incoming corpus/queries
+    # corpus = ['bob is in the house', 'susi goes to school', 'et tu brutu']
+    # queries = ['where is the school', 'bob is in the house']
+
+    tfidf_vectorizer = TfidfVectorizer()
+    tfidf_vectorizer.fit(corpus, queries)
+    corpus_vector = tfidf_vectorizer.transform(corpus)
+    queries_vector = tfidf_vectorizer.transform(queries)
+
+    similarity = cosine_similarity(queries_vector, corpus_vector)
```

Comment:
1) leave it running over night, its not our code, can't optimize it really
2) i have a KNN algorithm in the pipeline which should be faster